header-only/awkward/utils.h

```diff
@@ -12,14 +12,17 @@
 #include <stdexcept>
 #include <stdint.h>
 
+
 namespace awkward {
 
   /// @brief Returns the name of a primitive type as a string.
   template <typename T>
   const std::string
   type_to_name() {
-    std::cout << "Type " << typeid(T).name() << " is not recognized." << std::endl;
-    return typeid(T).name();
+    if(std::is_same_v<T, long long>) {
+    	return "int64";
```

Comment:
We may want to consider extending this to cover other ROOT type aliases if users request it: https://root.cern/doc/master/RtypesCore_8h_source.html